Data-Transfer-ReactJS-Dashboard/src/config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "DATA_TRANSFER_PROGRESS": {
+        "CLOCK": "http://ec2-35-88-36-61.us-west-2.compute.amazonaws.com:3000/d-solo/vUMFoZrnk/hls-data-transfer-metrics?orgId=1&refresh=5s&panelId=20",
```

Comment:
http? no https? would also be good to have just `//` (if we support both or port forwarding) so that we don't have cors issues.

Reply:
HTTPS isn't enabled yet Iksha. we will be enabling it during the last phase of work.

Reply:
That's why I was talking about `//` rather than specifying the schema.